### PR TITLE
feat(connector): add general lndhub connector #576

### LINKED ISSUE #576: Provide a LNDHub connector without BlueWallet brand
In getalby.com/login we give users the possibility to import the Lightning Address account into the Alby extension using the LNDhub connector. Currently this connector is branded with BlueWallet. Now I would like to see a neutral LNDHub connector without a BlueWallet branding. That might be less confusing for the user.

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

- Added general LNDHub connector (internally named lndhub-go)
- Reworked the lndhub ocnnector to handle BlueWallet and the new one

### Link this PR to an issue

Fixes #576

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes

- 
- 
- image

### How has this been tested?

- Connected an Alby account
- Connected an Bluewallet account

### CODE REVIEW

**Comment by MoritzKa** [comment]
The icon of the LNDhub connector has a different size than the others on the screenshot.

**Comment by MoritzKa** [comment]
Can you try one of these?
Slice 8
Slice 8 (1)

**Comment by escapedcat** [comment]
@MoritzKa updated it:
image

**Comment by MoritzKa** [comment]
Thanks. Looks good.

**Comment by bumi** [comment]
now we have 2 times "LndHub" written there. 
We should call the other one then only BlueWallet and also set the account name there then to BlueWallet. 

I would also move BlueWallet behind LNBits. BlueWallet's lndhub causes many problems as.

**Comment by escapedcat** [comment]
Done
image

**Review by im-adithya — APPROVED** [review]

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -110,13 +110,28 @@
...
+          "label": "BlueWallet Export URI"
+        },
+        "errors": {
+          "invalid_uri": "Invalid BlueWallet URI",
+          "connection_failed": "Connection failed. Is your BlueWallet URI correct?"
+        }
+      },
+      "lndhub_go": {
+        "title": "LNDHub",
+        "description": "Connect to your LNDHub mobile wallet",
```
Comment (bumi) [inline src/i18n/locales/en/translation.json:130]:
```suggestion
        "description": "Connect to your LNDHub account",
```

LNDHub is not a mobile wallet.

**Pull request merged**